SignalMessaging/profiles/OWSProfileManager.m

```diff
@@ -118,19 +56,12 @@ @interface OWSProfileManager ()
 @property (nonatomic, readonly) OWSIdentityManager *identityManager;
 
 // This property can be accessed on any thread, while synchronized on self.
-@property (nonatomic, readonly) UserProfile *localUserProfile;
-// This property can be accessed on any thread, while synchronized on self.
-@property (atomic, nullable) UIImage *localCachedAvatarImage;
```

Comment:
Just use `profileAvatarImageCache` to cache all profile avatars, including the local avatar.  Simpler.